examples/sample-layers/package.json

```diff
@@ -4,10 +4,13 @@
   },
   "dependencies": {
     "babel-polyfill": "^6.16.0",
-    "deck.gl": "3.1.0-beta.16",
+    "deck.gl": "^4.0.0-beta.2",
+    "extrude-polyline": "^1.0.6",
     "immutable": "^3.8.1",
-    "luma.gl": "3.0.0-alpha.4",
+    "lodash.flattendeep": "^4.4.0",
+    "luma.gl": "3.0.0-beta.9",
     "react": "^15.4.1",
+    "react-autobind": "^1.0.6",
```

Comment:
BTW - this module is unmaintained and has [some issues](https://github.com/cassiozen/React-autobind/issues/3)

I think most people are rolling w/ `core-decorators` `autobind` instead